docs/reference/provisioning/scim-via-curl.md

```diff
@@ -159,12 +159,10 @@ curl -X GET \
 
 ### get all users
 
-```bash
-curl -X GET \
-  --header "Authorization: Bearer $SCIM_TOKEN" \
-  --header 'Content-Type: application/json;charset=utf-8' \
-  $WIRE_BACKEND/scim/v2/Users/
-```
+There is a way to do this in the SCIM protocol, but it's not
+implemented in wire for performance reasons.  If you need a complete
```

Comment:
```suggestion
implemented in wire for performance reasons. If you need a complete
```

Reply:
Ok, I'm old, fine.  :)  https://www.thesaurus.com/e/writing/is-there-1-space-or-2-after-a-period/